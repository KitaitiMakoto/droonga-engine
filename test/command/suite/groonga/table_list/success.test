{
  "type": "table_create",
  "dataset": "Default",
  "body": {
    "name"  : "User",
    "flags" : "TABLE_PAT_KEY"
  }
}
{
  "type": "table_list",
  "dataset": "Default",
  "body": {
    "table" : "User"
  }
}
