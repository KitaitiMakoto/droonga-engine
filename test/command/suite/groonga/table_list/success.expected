{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "table_create.result",
  "body": [
    [
      0,
      0.0,
      0.0
    ],
    true
  ]
}
{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "table_list.result",
  "body": [
    [
      0,
      0.0,
      0.0
    ],
    [
      [
        [
          "id",
          "UInt32"
        ],
        [
          "name",
          "ShortText"
        ],
        [
          "path",
          "ShortText"
        ],
        [
          "flags",
          "ShortText"
        ],
        [
          "domain",
          "ShortText"
        ],
        [
          "range",
          "ShortText"
        ],
        [
          "default_tokenizer",
          "ShortText"
        ],
        [
          "normalizer",
          "ShortText"
        ]
      ],
      [
        256,
        "User",
        "/path/to/table",
        "TABLE_PAT_KEY|PERSISTENT",
        "ShortText",
        null,
        null,
        null
      ]
    ]
  ]
}
